code/src/java/plugin/exporttokens/TextToken.java

```diff
@@ -97,11 +98,11 @@ public String getToken(String tokenSource, PlayerCharacter pc, ExportHandler eh)
 
 		if (action.equalsIgnoreCase("UPPER") || action.equalsIgnoreCase("UPPERCASE"))
 		{
-			retString = retString.toUpperCase();
+			retString = retString.toUpperCase(Locale.ENGLISH);
```

Comment:
This is controlling what is in an output sheet - why do we believe it would be English?

Reply:
you're correct. I had intended for this to be part of my next round of adding explicit default locale calls. Fixed.